src/ucs/sys/iovec.inl

```diff
@@ -103,6 +111,16 @@
         __total_length; \
     })
 
+/**
+ * Calculates the offset in the IOV array that means how much data has
```

Comment:
i'd suggest to rename the function to "ucs_iov_iter_flat_offset"

Calculates the flat offset in the IOV array, which is the total data size before the position of the iterator.